clustering/singleton/extension/src/main/java/org/wildfly/extension/clustering/singleton/ElectionPolicyResourceDefinition.java

```diff
@@ -81,6 +81,7 @@ static PathElement pathElement(String value) {
 
         Attribute(String name, String alternative, CapabilityReferenceRecorder reference) {
             this.definition = createBuilder(name, alternative)
+                    .setAllowExpression(false)
```

Comment:
@bstansberry Can you confirm that disallowing expressions where previously enabled does not require model version bump (given there is nothing to transform)?